README.md

```diff
@@ -88,11 +88,23 @@ instances:
 
 During the Dyno start up, your YAML files are copied to the appropriate Datadog Agent configuration directories.
 
+## Limiting Datadog's Console Output
+
+When first set up, the Datadog agent may be logging more information to your application's console than you need. 
```

Comment:
This has a bit of a negative feel to it. Let's make it more neutral and combine with the sentence below:

To limit the logging output of the buildpack, set the `DD_LOG_LEVEL`...

Also, remove the "Note: In most cases..."